feature-libs/quote/components/details/overview/quote-details-overview.component.ts

```diff
@@ -5,24 +5,181 @@
  */
 
 import { Component } from '@angular/core';
-import { Quote, QuoteActionType, QuoteFacade } from '@spartacus/quote/root';
-import { TranslationService } from '@spartacus/core';
-import { Card } from '@spartacus/storefront';
-import { Observable } from 'rxjs';
-import { map } from 'rxjs/operators';
+import {
+  Quote,
+  QuoteState,
+  QuoteAction,
+  QuoteActionType,
+  QuoteDetailsReloadQueryEvent,
+  QuoteMetadata,
+  QuoteFacade,
+} from '@spartacus/quote/root';
+import { EventService, TranslationService } from '@spartacus/core';
+import {
+  Card,
+  FocusConfig,
+  ICON_TYPE,
+  SaveCardEvent,
+} from '@spartacus/storefront';
+import { combineLatest, Observable } from 'rxjs';
+import { map, take } from 'rxjs/operators';
 
 @Component({
   selector: 'cx-quote-details-overview',
   templateUrl: './quote-details-overview.component.html',
 })
 export class QuoteDetailsOverviewComponent {
   quoteDetails$: Observable<Quote> = this.quoteFacade.getQuoteDetails();
+  iconTypes = ICON_TYPE;
+  saveMode = false;
 
   constructor(
     protected quoteFacade: QuoteFacade,
+    protected eventService: EventService,
     protected translationService: TranslationService
   ) {}
 
+  focusConfig: FocusConfig = {
+    trap: false,
+    block: true,
+    autofocus: 'input',
+    focusOnEscape: false,
+  };
+
+  protected defineEmptyQuoteMetaData(): QuoteMetadata {
+    return {
+      description: undefined,
+      expirationTime: undefined,
+      name: undefined,
+    };
+  }
+
+  protected defineQuoteMetaData(saveCardEvent: SaveCardEvent): QuoteMetadata {
+    const metaData: QuoteMetadata = this.defineEmptyQuoteMetaData();
+    const metaDataPropertyNames = Object.getOwnPropertyNames(metaData);
+    const saveCardEventPropertyNames =
+      Object.getOwnPropertyNames(saveCardEvent);
+
+    metaDataPropertyNames?.forEach((name) => {
+      const index = saveCardEventPropertyNames.findIndex((element) =>
+        element.includes(name + '_')
+      );
+      if (index >= 0) {
+        const propertyName = saveCardEventPropertyNames[index];
+        metaData[name] = saveCardEvent[propertyName];
+      }
+    });
+    return metaData;
+  }
```

Comment:
As already mentioned here: https://github.com/SAP/spartacus/pull/17665/files#r1282305889 we should try to define and fill the saveCardEvent in a way, that we can easily extract/build the quote meta data.